src/view/app/Components/Layout/Query/QueryFormFooter.tsx

```diff
@@ -79,12 +79,15 @@ const QueryFormFooter: React.FC<QueryFormFooterProps> = ({
             >
                 <Grid item>
                     <MonospaceTypography>
-                        Records in grid:{' '}
-                        <MonospaceTypography
-                            sx={{ color: recordColor, display: 'inline-block' }}
+                        Records in grid:
```

Comment:
this should've been solved like this: `<MonospaceTypography component="span">`. Changing to `<span>` you lose the styles added above. Also, prefer to use MUI components instead of plain html, because they are styled by default and include applied themes.